compiler/damlc/tests/daml-test-files/ModuleImports_Main.daml

```diff
@@ -0,0 +1,9 @@
+-- Copyright (c) 2021, Digital Asset (Switzerland) GmbH and/or its affiliates.
+-- All rights reserved.
+
+-- This test checks that we produce an $$imports value including all modules from which this one imports instances.
+-- @QUERY-LF .modules[] | select(lf::get_dotted_name($pkg) == ["ModuleImports_Main"]) | .values[].name_with_type | select(lf::get_value_name($pkg) == ["$$$$imports"]) | lf::norm_imports($pkg) | any(. == {package: null, module: ["ModuleImports_OrphanInstance"]})
```

Comment:
Also this one, I don't think there's a way to split these into multiple lines, right?

Reply:
btw @cocreature, I did need the `select` on `ModuleImports_Main`, since otherwise I got all three `$$imports` definitions.

Reply:
oh right I was talking BS. sorry for the confusion.

And yes there is no way to split it into different lines atm.

Reply:
no problem, just thought you'd like to know for the future :) 

Got it, guess single line works for now, it's just a bit hard to read